src/main/resources/db/postgresql/V1_46__2491-RemoveMemberFromMpamCorrespondentTypeSelect.sql

```diff
@@ -0,0 +1,3 @@
+-- add a new column determining whether the frontend should show the case type as an option in the
+-- "Record Correspondent Details" dropdown
+ALTER TABLE case_type_correspondent_type ADD COLUMN selectable_in_dropdown boolean DEFAULT TRUE;
```

Comment:
Not sure on mentioning dropdown in the column name; feels like the database shouldn't be aware of the frontend behaviours.

Reply:
I have addressed this